tools/c7n_gcp/tests/data/events/k8s_create_cluster.json

```diff
@@ -0,0 +1,657 @@
+{
+    "protoPayload": {
+      "@type": "type.googleapis.com/google.cloud.audit.AuditLog",
+      "authenticationInfo": {
+        "principalEmail": "system:node:gke-standard-cluster-1-pool-1-c733eefe-xtbq"
+      },
+      "authorizationInfo": [
+        {
+          "granted": true,
+          "permission": "io.k8s.core.v1.nodes.create",
+          "resource": "core/v1/nodes/gke-standard-cluster-1-pool-1-c733eefe-xtbq"
+        }
+      ],
+      "methodName": "io.k8s.core.v1.nodes.create",
+      "request": {
+        "@type": "core.k8s.io/v1.Node",
+        "apiVersion": "v1",
+        "kind": "Node",
+        "metadata": {
+          "annotations": {
+            "volumes.kubernetes.io/controller-managed-attach-detach": "true"
+          },
+          "creationTimestamp": null,
+          "labels": {
+            "beta.kubernetes.io/arch": "amd64",
+            "beta.kubernetes.io/fluentd-ds-ready": "true",
+            "beta.kubernetes.io/instance-type": "n1-standard-1",
+            "beta.kubernetes.io/os": "linux",
+            "cloud.google.com/gke-nodepool": "pool-1",
+            "cloud.google.com/gke-os-distribution": "cos",
+            "failure-domain.beta.kubernetes.io/region": "us-central1",
+            "failure-domain.beta.kubernetes.io/zone": "us-central1-a",
+            "kubernetes.io/hostname": "gke-standard-cluster-1-pool-1-c733eefe-xtbq"
+          },
+          "name": "gke-standard-cluster-1-pool-1-c733eefe-xtbq"
+        },
+        "spec": {
+          "providerID": "gce://mythic-tribute-232915/us-central1-a/gke-standard-cluster-1-pool-1-c733eefe-xtbq"
+        },
+        "status": {
```

Comment:
please remove the status key in request and response, its extraneous and extremely large.

Reply:
Updated